### ISSUE TITLE
Mint 20 Beta looping logon

### ISSUE THREAD

**Issue by madrbperu** [linuxmint/mint20-beta#177]
Hello, I was trying to copy from one usb stick to another one using two different usb ports on a standalone tower PC Intel P4 2.4gz and 3gb ram all of the sudden the PC just shutdown, I went to restart it, logon screen through normal process but as soon as I entered my password it would take me back to the logon screen over and over, it was actually looping, it acted like it was going to bring me to desktop but then again, no luck, fortunately I had setup timeshift and picked a day before the event because the same day recovery file wouldn't do. It will be a bit hard to replicate given the special  circumstances, besides I can tell you that I picked up where I had left off with my copying with the same two usb sticks and hasn't happened again, just imagine a sudden shut down and not being able to get back in under this or any other event, there should be an automatic system-side memory image for these cases that will allow a recover up until a desktop is safe mode-like or a warning of instability getting to the desktop so the user can do what it takes to fix the problem immediately or at least back up their important files.

**Comment by NikoKrause** [comment]
I've never heard of a Mint installation, which does a shutdown on its own. But the looping login sounds familiar. Where do you store your timeshift snapshots? On a n external drive or do you use the same internal drive as for your Mint installation? Have you seen a warning that disk space is getting low before this incident?

**Comment by clefebvre** [comment]
No space left on device could prevent graphical login. If it happens again, drop to console (CTRL+ALT+F2) and login in there. Then check the space on HDD with df -u, try to login graphically (CTRL+ALT+F7), and when it fails, go back to console and check ~/.xsession-errors.

**Comment by clefebvre** [comment]
It doesn't explain why it shut down all of a sudden. Afaik even a kernel panic wouldn't do that. A power cut maybe, a lose cable? It's hard to say :)

**Comment by madrbperu** [comment]
Hello,

I really do appreciate your valuable time responding to my email, and since time is of the essence let's go to the heart of the matter, I read all the previous responses concerning this issue. To tell you the truth, it happened again but this time no USB involved, you are right about the very low space on the HDD, I downloaded the beta version of Linux Mint just for that purpose, testing.
Now, I have just gotten into Linux very recently, I am a fast learner despite my age (63) and I especially like Mint because I believe it's a very good system and I truly want to use it as my to-go OS and I want to feel happy about it using it.
I followed your advice from the other messages and didn't take me anywhere except finding out the low HDD space left, I will throw a few questions and comments at the end but these are my findings.
I shut the PC normally last night and today I bumped into this matter again, lucky for me you had already responded so I used a couple of your tricks, I bypassed the GUI logon screen using the KB key combinations but then I got stuck there trying to read the xsession-errors file and just couldn't find such file, even as root using sudo -s
I tried to get back to desktop and hit the logon screen even after I had logged in under cli so well, it was still looping, then I went back to the command line and guess what, I typed startx and brought me straight to desktop BUT it wasn't me as the user shown, it had the original background and layout, I know that because I had made some changes to my desktop, customized it with backgrounds and colorful icons so I think I was on root, anyhow I ran disk space analyzer and showed me I was indeed out of space with one snapshot of 13GB in a 30 GB IDE HDD and of course  the res was system files, now that I'm on the GUI I can see the /.xsession-errors file, should it matter anyway for the system to lock me out?, Or why let the system continue filling up until there's not a bite left? I know the user gets warnings but I think the system should halt and force the user to do some clean up by not allowing him/her to take another step except for deleting and/or uninstalling unused programs, etc.

Now, I logged out from root apparently to try to login as me through the CLI, not such luck, it tells me Fatal server error, could not write to pid to lock file, unable to connect to X server, unable to write authority file, giving up, connection refused.
I have windows experience and with some little logic, the problem could be in corrupted user profile, I have seen it before but with Microsoft but of course, this is Linux but still I sense the similarity.

Conclusion:

It has happened twice already, once it shutdown by itself and now the normal way, this is scary.
Second, this is a very special occurrence since I can only find on the web a few cases being mentioned about but still, there should be a failsafe kind of service for extreme things like these without discouraging people, simple users, from using Linux.
Third, I am in South America, in Peru and the field is practically virgin here on the usage of Linux and I was thinking in popularizing it by offering installations to Windows MS users that have had it with that OS, I believe I could change the mentality a bit and use open source software to benefit everyone, Linux Mint and people here too thus helping expand the goodness of the super solid OS like Linux itself.

My closing line could be just to wish we had the very best system ever put together to unite the World and give credit where credit is due if this and all issues get sorted out before the launching of the official release.

I will attach the /.xsession-errors files here for you to see and check, it is only 17kb

I will really appreciate if this and other matters are seen with efficiency to scream Free World...!

Thank you.


Manuel A. del Río Bartra
Administrador de Servicios Web
Cel: 976-678-998

[https://n7lpeq.bn1301.livefilestore.com/y2pORGVpGmLJOHcMH123M9KoP3djZkXY5e5P044LqN8X35pUXVdW6T8dTxhOf9L3LczJ-zoeg38SPW4hZ5dz5TihzrVWzABvN8Kat2f1n_ZfK0/cara1.jpg?psid=1]
________________________________
